torch/package/package_importer.py

```diff
@@ -337,20 +393,56 @@ def _make_module(
 
         return module
 
+    def _check_if_viable_parent(self, cur, child, name):
+        if isinstance(cur, _SelectiveExternNode):
```

Comment:
Please add a comment why this is skipped, it's non-obvious from looking at this function locally. 

Actually, it's not documented anywhere that `_SelectiveExternNode` has both explicit children (which are the interned modules, tracked in `self.children`) and implicit children (everything else, which are externed).

Reply:
By default a we expect all submodules of a selective extern module to be externed (ie. if `torch` is selective extern then `torch.nn` is externed). Therefore, a selective extern node is a valid parent of all modules whether they're externed or interned. Furthermore, only top level modules can be selective extern modules, so a selective extern module cannot be a child of anything that is not a root.